l10n_fr_department/model/res_company.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2014-2022 Akretion France (http://www.akretion.com/)
+# author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    country_department_id = fields.Many2one(
+        string="Country Department",
```

Comment:
As it is a related field, it will automatically take the string from the field it points to, so no need to redefine it.

Reply:
indeed ! removed. thanks !